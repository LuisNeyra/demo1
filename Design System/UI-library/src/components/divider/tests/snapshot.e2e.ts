import { snapIt } from '../../../test';

const component = 'divider';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('correctly', '<blaze-divider></blaze-divider>');
    snap('content', '<blaze-divider>Hello, world</blaze-divider>');
  });
});
