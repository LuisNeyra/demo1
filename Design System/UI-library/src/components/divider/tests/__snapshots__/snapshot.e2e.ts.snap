// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`divider renders content 1`] = `"<blaze-divider class=\\"hydrated\\"><!----><div class=\\"c-divider c-divider--solid\\"><span class=\\"c-divider__content\\">Hello, world</span></div></blaze-divider>"`;

exports[`divider renders correctly 1`] = `"<blaze-divider class=\\"hydrated\\"><!----><div class=\\"c-divider c-divider--solid\\"><span class=\\"c-divider__content\\"></span></div></blaze-divider>"`;
